src/pages/api/og/gallery/[galleryId].tsx

```diff
@@ -29,23 +29,10 @@ const handler = async (req: NextApiRequest) => {
       return fallbackImageResponse;
     }
 
-    // TODO(Rohan): remove these once we can support these assets
-    // temp fix to get the WLTA winner gallery frames working
-    const tempIgnoreTokensWithIds = new Set([
-      '2bT2G4iiB0LfMVZ6k3YfdiIs8sU',
-      '2bT2FzE3iB59Zm5PTUTAhM9lor7',
-      '2blxlBBmty8MFX3qLevWqOXorJX',
-      '2bhcj7DcaxAROSHodJH99glBt17',
-      '2cPoZ0hrNJbaiNsMOvLkeHrfIFc',
-      '2bT2FzEWNvgShbVLFwWDMnZ36ud',
-      '2cPoYvcYW2xsDoSHhJn9YoMFjY2',
-    ]);
-
```

Comment:
this is fixed now
<img width="1211" alt="Screenshot 2024-05-07 at 20 41 14" src="https://github.com/gallery-so/opengraph/assets/49758803/d954a037-8cb5-4606-89b6-2be66b19bf34">